### PR TITLE
Tanzu Edge Solutions Architecture - initial commit

### PR DESCRIPTION
**Pull request by bserratrice**
Initial commit for the release of the Tanzu Edge Solutions Architecture.

### CODE REVIEW

**Comment by craigtracey** [comment]
@bserratrice were you able to apply the changes I provided? Please reach out if you are having any issues.

**Comment by bserratrice** [comment]
> @bserratrice were you able to apply the changes I provided? Please reach out if you are having any issues.

Changes have been applied. Please review.

File: src/reference-designs/tkg-edge.md [hunk 1]
```diff
@@ -0,0 +1,885 @@
...
+- Ensuring security and compliance of edge environments.
+
+On top of the VMware Edge Compute Stack, the VMware Tanzu Edge Solution Architecture comprises core VMware Tanzu capabilities, including:
+
+- Unified Kubernetes Runtime: VMware Tanzu Kubernetes Grid (TKG) provides a consistent, upstream-compatible implementation of Kubernetes, that is tested, signed, and supported by VMware.
+- Global Multi-cluster Management: VMware Tanzu Mission Control (TMC) is a centralized management platform for consistently operating and securing your Kubernetes infrastructure and modern applications across multiple teams and clouds, regardless of where they reside.
+- Full-stack Observability: VMware Tanzu Observability by Wavefront (TO) is a high-performance streaming analytics platform that supports 3D observability (metrics, histograms, traces/spans). You can collect data from many services and sources across your entire application stack, including edge sites.
+
+This solution architecture is intended for anyone responsible for designing, implementing, managing and securing edge locations, including DevOps admins, VI admins, platform operators, and security teams.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:23]: ... managing, and securing ...

File: src/reference-designs/tkg-edge.md [hunk 2]
```diff
@@ -0,0 +1,885 @@
...
+
+VMware Tanzu Observability by Wavefront addresses the concerns mentioned above and significantly enhances observability. This SaaS offering collects and displays metrics and trace data from the full stack platform, as well as from applications. It provides the ability to create alerts tuned by advanced analytics, assist in the troubleshooting of systems, and help you understand the impact of running production code.
+
+The metrics can originate from your infrastructure or application. Kubernetes clusters at edge locations can install the Tanzu Observability collector as an extension in TKG to provide visibility into cluster operation and workload characteristics. This helps operators ensure a large fleet of clusters remain healthy, and enables rapid response when health characteristics change.
+
+Tanzu Observability is operated following a SaaS model, so outbound internet connectivity is required at edge sites.
+
+![](./img/tkg-edge/b24ac18e57b053174782c87492b3644d.png)
+
+Figure: Edge Kubernetes Cluster fleet observability
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:844]: Move into square brackets in line 842.

File: src/reference-designs/tkg-edge.md [hunk 3]
```diff
@@ -0,0 +1,885 @@
...
+
+Kubernetes clusters at edge locations can install the Fluent-bit logging extension and send cluster log data to a centralized log store, such as a vRealize Log Insights appliance. This allows for granular troubleshooting and analysis to be conducted from a centralized and searchable log indexer.
+
+vRealize Log Insight appliance is available as a separate on-prem deployable product. You can also choose to go with the SaaS version vRealize Log Insight Cloud.
+
+Fluent Bit is particularly well suited for Edge use case with its [buffering][802] capability that can be used to retain logs until the main datacenter reconnect.
+
+## Summary
+
+Tanzu Kubernetes Grid on top of the VMware Edge Compute stack provides an efficient way to deploy and manage Kubernetes infrastructure at the Edge at scale. Integrated with Tanzu Mission Control and Tanzu Observability, it gives company a way to focus on developing and deploying applications and feature for their customers rather than managing infrastructure.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:860]: .... it allows enterprises to ... applications and features for ...

File: src/reference-designs/tkg-edge.md [hunk 4]
```diff
@@ -0,0 +1,885 @@
...
+
+## Container Registry
+
+### Main Datacenter
+
+[Harbor Container Registry](https://goharbor.io/) can be deployed to both the centralized datacenter as well as to each edge location.  Using Harbor, replications can be configured such that all container images which get pushed into the centralized harbor are automatically replicated to each edge. Each edge deployment can reference its local harbor for efficient network resource utilization.
+
+![](./img/tkg-edge/tkg-edge-harbor.drawio.png)
+
+Figure: Harbor registry federated topology
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:418]: Global: Please move Figure headings into the square brackets for the figure.

File: src/reference-designs/tkg-edge.md [hunk 5]
```diff
@@ -0,0 +1,885 @@
...
+![](./img/tkg-edge/0547aae877598d9975f066451cb5b268.png)
+Figure: Harbor registry webhooks for edge site continuous delivery
+
+### At the Edge
+
+| Decision ID | Design Decision | Design Justification | Design Implication |
+| --- | --- | --- | --- |
+| `TZEDG-EDGE-REG-01` | Use Pull | more resilient, since the edge Harbors can pull in their own time, and it keeps configuration simple | configuration needs to be done at the edge site which might implies |
+
+Table: Design Decisions
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:437]: Move to line 432. Write as a lead, for example: The following table describes the justification and implication of the design decisions in the architecture.

File: src/reference-designs/tkg-edge.md [hunk 6]
```diff
@@ -0,0 +1,885 @@
+# VMware Tanzu Edge Solution Architecture
+
+Edge computing is gaining a lot of momentum for a variety of reasons that range from avoidance of latency, data volume, and bandwidth considerations to need for autonomous disconnected operation, privacy and security. These reasons are pushing businesses across all industries to run more and more applications at edge locations. Traditionally, these applications were run in virtual machines, but with emergence of Kubernetes and microservices architectures, more applications are being containerized and there is a need for running a cloud native compute stack at the edge.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:3]: ... to the need for autonomous ... operations, privacy, and ... Traditionally, these applications are ... run on VMs, ... but with

File: src/reference-designs/tkg-edge.md [hunk 7]
```diff
@@ -0,0 +1,885 @@
...
+
+For those companies that already have remote offices and branch offices with a few servers running edge workloads, running modern, distributed applications across your edge locations can add a new layer of complexity. There is a significant difference between running a cloud native platform at the core data center versus running it at edge locations.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:5]: For companies ...

File: src/reference-designs/tkg-edge.md [hunk 8]
```diff
@@ -0,0 +1,885 @@
...
+
+The VMware Edge Compute Stack helps companies manage their edge locations with little or no local IT staff and enables them to run VM workloads at these sites. Now with the addition of VMware Tanzu, you can simplify how you architect and deploy a cloud native stack at hundreds or thousands of edge locations so that you can run cloud native applications at the edge.
+
+The Tanzu Design for VMware Edge Compute Stack described here offers a best practice for running Tanzu at edge locations where the VMware Edge Compute Stack is already deployed. We’ll specifically address these challenges at the edge:
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:9]: The Tanzu design ...deployed. We will specifically address the following challenges ...

File: src/reference-designs/tkg-edge.md [hunk 9]
```diff
@@ -0,0 +1,885 @@
...
+- Running and managing Kubernetes infrastructure
+- Providing a consistent development and deployment environment for applications across the cloud, datacenter, and edge
+- Managing a fleet of Kubernetes clusters
+- Proactively monitoring clusters and apps
+- Ensuring security and compliance of edge environments.
+
+On top of the VMware Edge Compute Stack, the VMware Tanzu Edge Solution Architecture comprises core VMware Tanzu capabilities, including:
+
+- Unified Kubernetes Runtime: VMware Tanzu Kubernetes Grid (TKG) provides a consistent, upstream-compatible implementation of Kubernetes, that is tested, signed, and supported by VMware.
+- Global Multi-cluster Management: VMware Tanzu Mission Control (TMC) is a centralized management platform for consistently operating and securing your Kubernetes infrastructure and modern applications across multiple teams and clouds, regardless of where they reside.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:20]: Global Multi-Cluster Management or Global multi-cluster management
Full-Stack Observability or Full-stack observability

File: src/reference-designs/tkg-edge.md [hunk 10]
```diff
@@ -0,0 +1,885 @@
...
+- Full-stack Observability: VMware Tanzu Observability by Wavefront (TO) is a high-performance streaming analytics platform that supports 3D observability (metrics, histograms, traces/spans). You can collect data from many services and sources across your entire application stack, including edge sites.
+
+This solution architecture is intended for anyone responsible for designing, implementing, managing and securing edge locations, including DevOps admins, VI admins, platform operators, and security teams.
+
+## Topology
+
+There are multiple topologies that can be leveraged for edge deployments. A centralized model, edge site are managed from a central hub (Hub and Spoke topology). A decentralized model is also available where each edge site is managed locally.
+This solution guide is focused on the centralized model where a vCenter is deployed at the main datacenter location and a datacenter with vSphere cluster at each edge location. It is assumed that the vSphere infrastructure is already deployed.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:28]: Replace lines 27 and 28 with the following - 
Multiple topologies are possible for edge deployments. Some of the topologies are described in this section. However, the VMware Tanzu Edge Solution Architecture uses a centralized or hub and spoke topology, where a vCenter is deployed at the main datacenter location and a vSphere cluster is deployed in the datacenter at each edge location. This design document assumes that the vSphere infrastructure is already deployed.

File: src/reference-designs/tkg-edge.md [hunk 11]
```diff
@@ -0,0 +1,885 @@
...
+An Edge location can be a storefront (point of sale), manufacturing facility, medical office, or other IT infrastructure located in a remote, distributed site. Although these locations are important to your organization, they usually run up to 100 tenant virtual machines. Because of the critical nature of the workloads, dependencies on a Wide Area Network (WAN) connection must be limited.
+
+### Hub and Spoke
+
+In this topology, the management plane is entirely residing in the main datacenter. The centralized management plane composed of vCenter, NSX Advanced Load Balancer controller and TKG Management cluster are all in the same central location.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:33]: ... the management plane resides entirely in ... NSX Advanced Load Balancer Controller, and Tanzu Kubernetes Grid management cluster ...

File: src/reference-designs/tkg-edge.md [hunk 12]
```diff
@@ -0,0 +1,885 @@
...
+Only the TKG Workload clusters are created at the edge along with Load Balancing components (Service Engines). This design provides a minimal footprint at the edge.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:34]: Only the Tanzu Kubernetes Grid workload clusters and the NSX Advanced Load Balancer Service Engines (SEs) are created at the edge.

Comment (srinivasme21) [reply src/reference-designs/tkg-edge.md:34]: Global: Use expanded product names - Tanzu Kubernetes Grid, Tanzu Mission Control, NSX Advanced Load Balancer, Tanzu Observability, Tanzu Service Mesh. 
Exception - Tables and figures. TKG, TMC, NSX ALB, TO, TSM.

File: src/reference-designs/tkg-edge.md [hunk 13]
```diff
@@ -0,0 +1,885 @@
...
+Once the Edge site is deployed, Tanzu Mission Control is used for fleet management, allowing an easy way to manage and lifecycle TKG workload clusters at the edge.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:35]: Once the edge site is ... Tanzu Kubernetes Grid workload ...

File: src/reference-designs/tkg-edge.md [hunk 14]
```diff
@@ -0,0 +1,885 @@
...
+Additional components for monitoring, backup and log management can also be present in the main datacenter.
+
+![](./img/tkg-edge/tkg-edge-hub-spoke.drawio.png)
+
+#### Key Features
+
+- Minimal footprint.
+- Limited recoverability when site is disconnected
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:43]: Add period.

File: src/reference-designs/tkg-edge.md [hunk 15]
```diff
@@ -0,0 +1,885 @@
...
+An Edge location can be a storefront (point of sale), manufacturing facility, medical office, or other IT infrastructure located in a remote, distributed site. Although these locations are important to your organization, they usually run up to 100 tenant virtual machines. Because of the critical nature of the workloads, dependencies on a Wide Area Network (WAN) connection must be limited.
+
+### Hub and Spoke
+
+In this topology, the management plane is entirely residing in the main datacenter. The centralized management plane composed of vCenter, NSX Advanced Load Balancer controller and TKG Management cluster are all in the same central location.
+Only the TKG Workload clusters are created at the edge along with Load Balancing components (Service Engines). This design provides a minimal footprint at the edge.
+Once the Edge site is deployed, Tanzu Mission Control is used for fleet management, allowing an easy way to manage and lifecycle TKG workload clusters at the edge.
+Additional components for monitoring, backup and log management can also be present in the main datacenter.
+
+![](./img/tkg-edge/tkg-edge-hub-spoke.drawio.png)
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:38]: Add text in square brackets. Possible text: Hub and spoke topology.

File: src/reference-designs/tkg-edge.md [hunk 16]
```diff
@@ -0,0 +1,885 @@
...
+
+#### Key Features
+
+- Minimal footprint.
+- Limited recoverability when site is disconnected
+- Quick and easy to deploy
+
+### Hybrid
+
+In this topology, only the TKG Management cluster is located in a central location.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:48]: ... Tanzu Kubernetes Grid management cluster ...

File: src/reference-designs/tkg-edge.md [hunk 17]
```diff
@@ -0,0 +1,885 @@
...
+Each Edge site has the infrastructure management plane (vCenter, and NSX Advanced Load Balancer controllers, DNS servers and NTP server). This design provides a better availability when the edge location gets disconnected from the main datacenter.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:49]: Each edge site contains the infrastructure management plane, which includes vCenter, NSX Advanced Load Balancer Controllers, DNS servers, and NTP servers. This design provides better availability ...

File: src/reference-designs/tkg-edge.md [hunk 18]
```diff
@@ -0,0 +1,885 @@
...
+Some additional components for monitoring, backup and log management could be also installed at the Edge location to provide these functions locally.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:50]: Additional component for monitoring, backup, and log management can also be installed at the edge location to provide the functionality locally.

File: src/reference-designs/tkg-edge.md [hunk 19]
```diff
@@ -0,0 +1,885 @@
...
+
+![](./img/tkg-edge/tkg-edge-hybrid.drawio.png)
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:52]: Suggested text to add in square brackets - Hybrid topology

File: src/reference-designs/tkg-edge.md [hunk 20]
```diff
@@ -0,0 +1,885 @@
...
+- Control Plane node:
+In case of a Kubernetes cluster with a single control plane node, the management of the cluster will be unavailable for few minutes until the control plane is restarted and all the Kubernetes components are back up and running.
+- Worker node with application pods and persistent volumes:
+If the application is deployed in a highly available fashion, there should not be any disruption.
+
+### Centralized control plane
+
+For companies modernizing their application estate, it is not uncommon to have cloud native deployments across many locations. They will run Kubernetes clusters in their core data centers (in the 10s), mini data centers away from the core data center (in the 100s), and thousands of smaller 2 to 3 server deployments at locations like stores, like what we are testing here. To be able to manage at scale this vast Kubernetes estate you need a centralized control plane that can help you deploy, manage, secure and provide insights into kubernetes deployments. For this reason, a centralized control plane in public cloud, private data center, or a hybrid environment is a requirement. One way to solve this is via Tanzu Mission Control.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:717]: You will run Kubernetes clusters in your core data centers (in the 10s), mini data centers away from the core data center (in the 100s), and thousands of smaller 2 to 3 server deployments at locations like stores, similar to this design. To manage this vast Kubernetes estate at scale, you need a centralized control plane that can help you deploy, manage, secure, and provide insights into your Kubernetes deployments.

File: src/reference-designs/tkg-edge.md [hunk 21]
```diff
@@ -0,0 +1,885 @@
...
+
+TMC is a centralized management platform for consistently operating and securing your Kubernetes infrastructure and modern applications across multiple teams and clouds. It can manage and ensure operations across a large fleet of edge Kubernetes clusters consistently with the capabilities described below
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:719]: Tanzu Mission Control is a centralized ... consistently with the following capabilities:

File: src/reference-designs/tkg-edge.md [hunk 22]
```diff
@@ -0,0 +1,885 @@
...
+
+In the Tanzu Mission Control resource hierarchy, you can specify policies at the following three levels:
+
+- Organization
+- Object groups (cluster groups and workspaces)
+- Kubernetes objects (clusters and namespaces)
+
+#### Global Visibility example
+
+With TMC you can group clusters based on different types of locations and manage fleets of clusters instead of managing them individually. When we are dealing with large scale edge deployments this becomes very critical as customers might have two or three different types of locations. Being able to group them with cluster groups, looking at individual fleet health, performing collective operations is key for edge deployments. Figure X shows an overview of the cluster group.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:756]: With Tanzu Mission Control ... The following figure shows an overview ...

File: src/reference-designs/tkg-edge.md [hunk 23]
```diff
@@ -0,0 +1,885 @@
...
+
+![](./img/tkg-edge/ce29c1e20bf0ecc1ffe2dfdc658cb585.png)
+
+Figure: Managing the edge cluster fleet using TMC
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:760]: Move the figure caption into the square brackets.

File: src/reference-designs/tkg-edge.md [hunk 24]
```diff
@@ -0,0 +1,885 @@
...
+
+#### Policy Management example
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:762]: Example

File: src/reference-designs/tkg-edge.md [hunk 25]
```diff
@@ -0,0 +1,885 @@
...
+
+Policies allow you to provide a set of rules that govern your organization and all the objects it contains. Policies can be applied to fleets of clusters by defining once to ensure that the kubernetes clusters and applications are secure, compliant while enabling self service which makes it very appealing for edge scale.  Tanzu mission control provides following policy types that can be applied to fleets of clusters:
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:764]: Policies allow ... ensure that the Kubernetes clusters and ... Tanzu Mission Control provides ...

File: src/reference-designs/tkg-edge.md [hunk 26]
```diff
@@ -0,0 +1,885 @@
...
+
+1. Access policy
+2. Image registry policy
+3. Network policy
+4. Quota policy
+5. Pod Security Policy.
+
+Figure X and Figure X below showcase how customers can create policies and use the policy insights page to review if any of the policies are violated across all the edge deployments.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:772]: The following figures showcase how customers ...

File: src/reference-designs/tkg-edge.md [hunk 27]
```diff
@@ -0,0 +1,885 @@
...
+
+![](./img/tkg-edge/a93e92ad27bee927a258c5447a3ed78d.png)
+
+Figure: Using TMC to manage edge cluster policies
+
+![](./img/tkg-edge/559bfe3fa044f89dab66d033ee2168cf.png)
+
+Figure: Policy reporting across all edge locations
+
+#### Usage consideration
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:782]: Considerations

File: src/reference-designs/tkg-edge.md [hunk 28]
```diff
@@ -0,0 +1,885 @@
...
+
+As of time of writing [Tanzu Mission Control][701] requires,  at least, ~1.5GB of memory and ~1 vCPU per managed cluster to operate.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:784]: Tanzu Mission Control requires, at least, ...

File: src/reference-designs/tkg-edge.md [hunk 29]
```diff
@@ -0,0 +1,885 @@
...
+2. Image registry policy
+3. Network policy
+4. Quota policy
+5. Pod Security Policy.
+
+Figure X and Figure X below showcase how customers can create policies and use the policy insights page to review if any of the policies are violated across all the edge deployments.
+
+![](./img/tkg-edge/a93e92ad27bee927a258c5447a3ed78d.png)
+
+Figure: Using TMC to manage edge cluster policies
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:776]: Move figure caption into the square brackets.

File: src/reference-designs/tkg-edge.md [hunk 30]
```diff
@@ -0,0 +1,885 @@
...
+
+![](./img/tkg-edge/559bfe3fa044f89dab66d033ee2168cf.png)
+
+Figure: Policy reporting across all edge locations
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:780]: Move figure caption into the square brackets.

File: src/reference-designs/tkg-edge.md [hunk 31]
```diff
@@ -0,0 +1,885 @@
...
+
+#### Usage consideration
+
+As of time of writing [Tanzu Mission Control][701] requires,  at least, ~1.5GB of memory and ~1 vCPU per managed cluster to operate.
+
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:785]: The following table describes the resources used by Tanzu Mission Control components.

File: src/reference-designs/tkg-edge.md [hunk 32]
```diff
@@ -0,0 +1,885 @@
...
+We recommend the following best practices for managing identities in Tanzu Kubernetes Grid provisioned clusters:
+
+- Configure Pinniped services during management cluster creation.
+- Limit access to cluster resources following the [least privilege](https://csrc.nist.gov/glossary/term/least_privilege) principle.
+- Limit access to management clusters to the appropriate set of users. For example, provide access only to users who are responsible for managing infrastructure and cloud resources but not to application developers. This is especially important because access to the management cluster inherently provides access to all workload clusters.
+- Limit cluster administrator access for workload clusters to the appropriate set of users. For example, provide access to users who are responsible for managing infrastructure and platform resources in your organization, but not to application developers.
+- Connect to an [identity provider](https://csrc.nist.gov/glossary/term/identity_provider) to manage the user identities allowed to access cluster resources instead of relying on administrator-generated `kubeconfig` files.
+
+## Monitoring, Logging and Observability
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:828]: ... Logging, and ...

File: src/reference-designs/tkg-edge.md [hunk 33]
```diff
@@ -0,0 +1,885 @@
...
+
+The metrics can originate from your infrastructure or application. Kubernetes clusters at edge locations can install the Tanzu Observability collector as an extension in TKG to provide visibility into cluster operation and workload characteristics. This helps operators ensure a large fleet of clusters remain healthy, and enables rapid response when health characteristics change.
+
+Tanzu Observability is operated following a SaaS model, so outbound internet connectivity is required at edge sites.
+
+![](./img/tkg-edge/b24ac18e57b053174782c87492b3644d.png)
+
+Figure: Edge Kubernetes Cluster fleet observability
+
+TKG also supports Grafana and Prometheus as alternative on premise solutions that can be used for monitoring.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:846]: Tanzu Kubernetes Grid ... as alternative on premises solutions ...

File: src/reference-designs/tkg-edge.md [hunk 34]
```diff
@@ -0,0 +1,885 @@
...
+
+### Log Aggregation
+
+vRealize Log Insight (vRLI) provides real-time log management and log analysis with machine learning based intelligent grouping, high-performance searching, and troubleshooting across physical, virtual, and cloud environments. vRLI already has a deep integration with the vSphere platform where you can get key actionable insights, and it can be extended to include the cloud native stack as well.
+
+Kubernetes clusters at edge locations can install the Fluent-bit logging extension and send cluster log data to a centralized log store, such as a vRealize Log Insights appliance. This allows for granular troubleshooting and analysis to be conducted from a centralized and searchable log indexer.
+
+vRealize Log Insight appliance is available as a separate on-prem deployable product. You can also choose to go with the SaaS version vRealize Log Insight Cloud.
+
+Fluent Bit is particularly well suited for Edge use case with its [buffering][802] capability that can be used to retain logs until the main datacenter reconnect.
```
Comment (srinivasme21) [inline src/reference-designs/tkg-edge.md:856]: ... for an edge use case ... reconnects.

**Review by srinivasme21 — CHANGES_REQUESTED** [review]
Global: Please replace screen captures with light background

**Review by srinivasme21 — CHANGES_REQUESTED** [review]
Global: Please replace screen captures with light background

**Review by srinivasme21 — CHANGES_REQUESTED** [review]
Global: Please replace screen captures with light background

**Comment by bserratrice** [comment]
> Global: Please replace screen captures with light background

The lab environment is not available anymore, I'll this in mind for the next major release.

**Pull request merged**